commands/link.js

```diff
@@ -0,0 +1,158 @@
+var Command = require('ronin').Command;
+var Q = require('q');
+var fs = require('fs-extra');
+var path = require('path');
+var readFile = Q.denodeify(fs.readFile);
+var readDir = Q.denodeify(fs.readdir);
+var remove = Q.denodeify(fs.remove);
+var inquirer = require('inquirer');
+var utils = require('../lib/util');
+var createLink = require('../lib/createLink');
+var chalk = require('chalk');
+
+module.exports = Command.extend({
+    help: function () {
+        var title = chalk.bold;
+        var d = chalk.gray;
+        var r = '\n  ' + title('Usage') + ': bb ' + this.name + ' [OPTIONS]';
+        r += '\n\t Symlinks source directory to target.\n\t  If --lp path is set, target will be: ';
+        r += '\n\t    ' +  d('{lp path}/launchpad-bundles/static/launchpad/{bundle}/widgets/{package name}');
+        r += '\n\t  If package name starts with `widget-` it will be stripped out.';
+        r += '\n\n\t  If --cxp path is set, target will be: ';
+        r += '\n\t    ' +  d('{cxp path}/src/main/webapp/static/widgets/{package name}');
+        r += '\n\n  ' + title('Options') + ': -short, --name <type> ' + d('default') + ' description\n\n';
+        r += '      -s,  --source <string>\t\t' + d('current directory') + '\t Path to source directory.\n';
+        r += '      -t,  --target <string>\t\t\t\t\t Path to directory in which to (un)link a source.\n';
```

Comment:
"in which to (un)link a source" - in this case we're only linking, and with other flag UNlinking. So it's better to remove this confusing "(un)".